packages/components/select/tests/select.test.tsx

```diff
@@ -101,3 +101,79 @@ describe("<Select />", () => {
     )
   })
 })
+
+describe("<MultiSelect />", () => {
+  test("MultiSelect renders correctly", async () => {
+    const { container } = render(
+      <>
+        <MultiSelect>
+          <Option value="one">One</Option>
+          <Option value="two">Two</Option>
+        </MultiSelect>
+      </>,
+    )
+    await a11y(container)
+  })
+
+  test("when items passed to props, renders correctly", async () => {
+    const { container } = render(
+      <MultiSelect
+        items={[
+          { label: "One", value: "One" },
+          { label: "Two", value: "Two" },
+        ]}
+      />,
+    )
+    await a11y(container)
+  })
+
+  test("when nested items passed to props, renders correctly", async () => {
+    const { container } = render(
+      <MultiSelect
+        items={[
+          {
+            label: "One",
+            items: [
+              { label: "Nested One-One", value: "Nested One-One" },
+              { label: "Nested One-Two", value: "Nested One-Two" },
+            ],
+          },
+          {
+            label: "Two",
+            items: [
+              { label: "Nested Two-One", value: "Nested Two-One" },
+              { label: "Nested Two-Two", value: "Nested Two-Two" },
+            ],
+          },
+        ]}
+      />,
+    )
+    await a11y(container)
+  })
+
+  test("when items clicked twice, renders correctly", async () => {
+    const { user } = render(
+      <MultiSelect data-testid="MultiSelect">
+        <Option data-testid="Option-One" value="one">
+          One
+        </Option>
+        <Option data-testid="Option-Two" value="two">
+          Two
+        </Option>
+        <Option data-testid="Option-Three" value="three">
+          Three
+        </Option>
+      </MultiSelect>,
+    )
+
+    const multiSelect = screen.getByTestId("MultiSelect")
+    await user.click(multiSelect)
+    const optionOne = screen.getByTestId("Option-One")
+    await user.click(optionOne)
+
+    const optionTwo = screen.getByTestId("Option-Two")
+    await user.click(optionTwo)
+
+    expect(screen.getByText("One,")).toBeInTheDocument()
+  })
```

Comment:
This test has issued a warning.
Refer to other tests.

Reply:
Thank you for your comment!
I fixed this commit. 
https://github.com/yamada-ui/yamada-ui/pull/935/commits/10cac9591567a684c714cc0d16db6344ed1e38f0